pykernels/graph/shortestpath.py

```diff
@@ -18,9 +18,9 @@ def floyd_warshall(adj_mat, weights):
     res = res + ((adj_mat != 0) * weights)
     res[res == 0] = np.inf
     np.fill_diagonal(res, 0)
-    for i in xrange(N):
-        for j in xrange(N):
-            for k in xrange(N):
+    for i in range(N):
```

Comment:
Since we are talking about one single file where the xrange functions were replaced, we could simply do this:
```python
try:
  range = xrange
except:
  pass
```